docs/network-policy.md

```diff
@@ -0,0 +1,144 @@
+# Cluster Network Policy
+
+ClusterNetworkPolicy is a specification of how workloads within a cluster
+communicate with each other and other external endpoints.
+The ClusterNetworkPolicy is supposed to aid cluster-admins to determine
+the security policy for the cluster, unlike K8s NetworkPolicy, which is
+aimed towards developers to secure their apps.
```

Comment:
maybe also mention that they only apply to Pods in a specific namespace

Reply:
done